drf_kit/tests.py

```diff
@@ -117,6 +117,14 @@ def assertResponseNotAllowed(self, response):
             response_key=None,
         )
 
+    def assertResponseAccepted(self, response: Response, expected_item: Any = ""):
```

Comment:
```suggestion
    def assertResponseAccepted(self, response: Response, expected_item: Any = None):
```

Setting the default to None will assure that [it always "ignores" the parameter](https://github.com/flamingo-run/drf-toolkit/blob/main/drf_kit/tests.py#L143).
Settings to `""` will fail unless the response is exactly this string, which is a hard assumption for most endpoints.

Reply:
The DRF returns an empty string when no `data` is provided. That's why I kept empty string instead of None

Reply:
I see.
Thinking about the most common use case, 202 responses have a content (204 is usually empty).
So this drf-kit helper should, by default to be friendly, don't validate the body, correct?
In order to do not validate the body, it must be None.

If you keep the default an empty string, all endpoints that return anything (the most common) will fail.

Reply:
If an endpoint returns nothing, it will return an empty string "". We always check the body, even when the expected_body is set to None. I didn't get how keeping the default value as None would work for default values.